addons/web/static/src/core/code_editor/code_editor.js

```diff
@@ -168,13 +148,5 @@ export class CodeEditor extends Component {
             },
             () => [this.props.sessionId, this.props.mode, this.props.value]
         );
-
-        const debouncedResize = useDebounced(() => {
```

Comment:
This is kinda odd according to Ace's doc

> Changing the size of the editor
> Ace only checks for changes of the size of it's container when window is resized. If you resize the editor div in another manner, and need Ace to resize, use the following:
> 
> editor.resize()
> if you want editor to change it's size based on contents, use maxLines option as shown in https://ace.c9.io/demo/autoresize.html

at https://ace.c9.io/#nav=howto